front-dispatcher/src/components/Header/FixedHeader/FixedHeader.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import Header from "./FixedHeader";
+import LogoSvg from "../../UI/LogoSvg/LogoSvg";
+import { storyStrings } from "../../../strings/storyStrings";
+import HeadrUserBtn from "../HeaderUserBtn/HeadrUserBtn";
+import { Container } from "../../UI/Container/Container";
+import { headerStrings } from "../../../strings/headerString";
+import SvgBtn from "../HeaderSvgs/svgBtn";
+
```

Comment:
I think the story is too long. It should be shorter and dynamic, is an example:
import React from "react";
import { ComponentStory, ComponentMeta } from "@storybook/react";

import Button from "./Button";

export default {
  title: "Components/Button",
  component: Button,
} as ComponentMeta<typeof Button>;

const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;

export const Button1 = Template.bind({});
Button1.args = {
  children: "יצירת אירוע",
  variant: "contained",
  buttonType: "primary",
  isLoading: false,
};